assembl/static2/js/app/components/administration/brightMirror/index.jsx

```diff
@@ -0,0 +1,127 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { I18n } from 'react-redux-i18n';
+import arrayMutators from 'final-form-arrays';
+import { type ApolloClient, compose, withApollo } from 'react-apollo';
+import { Field } from 'react-final-form';
+
+// Components
+import AdminForm from '../../form/adminForm';
+import FileUploaderFieldAdapter from '../../form/fileUploaderFieldAdapter';
+import MultilingualTextFieldAdapter from '../../form/multilingualTextFieldAdapter';
+import MultilingualRichTextFieldAdapter from '../../form/multilingualRichTextFieldAdapter';
+import LoadSaveReinitializeForm from '../../../components/form/LoadSaveReinitializeForm';
+import Navbar from '../navbar';
+import Loader from '../../common/loader';
+import Helper from '../../common/helper';
+
+// Functions
+import { load, postLoadFormat } from './load'; // Load file needs to be updated according to bright mirror requirements
+import { createMutationsPromises, save } from './save'; // Save file needs to be updated according to bright mirror requirements
+import validate from './validate'; // Save file needs to be updated according to bright mirror requirements
+
+type Props = {
+  currentStep: number,
+  client: ApolloClient,
+  editLocale: string
+};
+
+const name = 'themes[0]'; // We have only one thematic for BrightMirror
+
+const loading = <Loader />;
+
+const BrightMirrorAdminForm = ({ client, currentStep, editLocale }: Props) => (
+  <LoadSaveReinitializeForm
+    load={(fetchPolicy: FetchPolicy) => load(client, fetchPolicy)}
+    loading={loading}
+    postLoadFormat={postLoadFormat}
+    createMutationsPromises={createMutationsPromises(client)}
+    save={save}
+    validate={validate}
+    mutators={{
```

Comment:
"you don't need arrayMutators if your form does not contain an array (FieldArray or FieldArrayWithActions component)"